client/src/pages/Feed.js

```diff
@@ -258,6 +260,41 @@ const Feed = () => {
     dispatchAction(TOGGLE_STATE, "showFilters");
   }
 
+  const handlePostLike = (postId) => {
```

Comment:
Let's use async-await syntax for this function to be consistent with the code-base.

Reply:
Done.